trace-mini-agent/src/trace_processor.rs

```diff
@@ -111,13 +123,18 @@ impl TraceProcessor for ServerlessTraceProcessor {
 
         // send trace payload to our trace flusher
         match tx.send(tracer_payload).await {
-            Ok(_) => Response::builder()
-                .status(200)
-                .body(Body::from("Successfully buffered traces to be flushed.")),
-            Err(e) => Response::builder().status(500).body(Body::from(format!(
-                "Error sending traces to the trace flusher. {}",
-                e
-            ))),
+            Ok(_) => {
+                info!("Successfully buffered traces to be flushed.");
+                let body =
+                    json!({ "message": "Successfully buffered traces to be flushed." }).to_string();
```

Comment:
do we need a body? who is reading it?

Reply:
the JS tracer will log a debug log with a "unable to parse JSON" error if the body is not a JSON string. It doesn't effect any functionality to not include a body however. I thought this might be confusing to users with debug logs turned on, so I added bodies